### ISSUE TITLE
Squished square result when taking screenshot from menu

### ISSUE THREAD

**Issue by dotcommer** [jrbudda/Vivecraft_116#41]
Not sure if I'm doing something wrong, or if its an issue with Vivecraft, Steam, or something else.  I noticed clicking on the in-game menu button "Screenshot", will save a square aspect ratio .png image, but it doesn't crop, it smooshes the entire view into that square.  It's a bummer.  Any ideas on how to fix that, if possible?  Thanks!

**Comment by Techjar** [comment]
That is just what your HMD view looks like. There is a camera you can pull out to take 16:9 screenshots if you want. The binding for it should be in the radial menu by default.

**Comment by dotcommer** [comment]
Hey Techjar, I tried the screenshot option in the radial menu, but it produces the same result as the "screenshot" menu option.  Am I missing something?  You mentioned a "camera" that I could "pull out", but didn't see that option in the radial menu.  It just seemed to take a screenshot and post the file link to it in chat.  I searched through all the other video and VR settings but couldn't find anything related to setting a custom resolution for screenshots (beyond just a multiplier of the desktop resolution).  All images are still squished into a 4:3 aspect ratio.
